src/training/lstmtraining.cpp

```diff
@@ -78,6 +78,8 @@ int main(int argc, char **argv) {
     tprintf("Must provide a --model_output!\n");
     return 1;
   }
+  if (CreateDirIfNotExists(FLAGS_model_output.c_str(), "model_path"))
+    return 1;
   if (FLAGS_traineddata.empty()) {
     tprintf("Must provide a --traineddata see training wiki\n");
     return 1;
```

Comment:
The following code declares `STRING model_output` and fills it with a purified model name, but that new name is never used. That looks wrong. Maybe the intention was to use `model_output` instead of `FLAGS_model_output`, but that was never finished.

Reply:
Should we remove the dead code, or should we use it?

Reply:
If we use code it not dead anymore ;-)